### PR TITLE
fix: update links to user configuration options in the docs

### PR DESCRIPTION
**Pull request by ivan-savciuc**
## About this change—what it does

Fix link in the docs to the user config options 
Resolves: #906.

### CODE REVIEW

File: docs/guides/syslog-service-integration.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Then, we define the `aiven_service_integration_endpoint`.
 
 The important things here are:
 - `endpoint_type` - The type of integration (e.g. `rsyslog`, `prometheus`, `Datadog`)
-- `{type}_user_config` - The user config contains the connection info for your endpoint, such as: URL, port, Certificates (as strings) and login info. {type} here is the endpoint_type you specified above. The documentation for these configs is [here](https://github.com/aiven/terraform-provider-aiven/tree/master/aiven/templates/integration_endpoints_user_config_schema.json) but generated documentation for this is coming soon.
+- `{type}_user_config` - The user config contains the connection info for your endpoint, such as: URL, port, Certificates (as strings) and login info. {type} here is the endpoint_type you specified above. The documentation for these configs is [here](https://github.com/aiven/aiven-go-client/blob/master/tools/exp/dist/integration_endpoint_types.yml) but generated documentation for this is coming soon.
```
Comment (byashimov) [inline docs/guides/syslog-service-integration.md:20]: This is tricky. I would probably refer to API docs, rather than to the client.

Comment (Serpentiel) [reply docs/guides/syslog-service-integration.md:20]: unfortunately, I believe this is documented in a tricky way in our API docs, so this makes it an actually good way to do it

**Review by Serpentiel — APPROVED** [review]
lgtm

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**